src/components/CartFilled.vue

```diff
@@ -8,13 +8,16 @@
     <div class="px-2">
       <img src="@/assets/images/image-product-1.jpg" alt="product-image" class="prd-img">
     </div>
-    <div>
+    <div class="px-2">
       <p> Fall Limited Edition Sneakers</p>
-      <p> $125.00 x {{}}</p>
+      <p> $125.00 x 3 <span class="font-bold">$375.00</span></p>
     </div>
+     <div class="px-2 text-left text-gray-300" @click="deleteCart">
+      <font-awesome-icon icon="fa-solid fa-trash" />
+      </div>
    
   </div>
-  <button class="m-4 px-10 py-3 bg-orange rounded-md text-white"> Checkout </button>
+  <button class="my-4 mx-8 px-10 py-3 w-3/4 bg-orange rounded-md text-white font-bold"> Checkout </button>
```

Comment:
[request] Can we have screenshots of the mobile view ?

Reply:
![image](https://user-images.githubusercontent.com/72354846/176711084-fe857f9e-4796-4971-b78e-2b6521920227.png)
![image](https://user-images.githubusercontent.com/72354846/176711271-7bdf2623-a553-439d-88fe-2bcc0d7a4207.png)